VnE/Source/Extensions/ChemPack/MoleculeClass.py

```
@@ -174,10 +173,9 @@ def __eq__(self, other):
             return False
 
     def __del__(self):
-        for i, p in enumerate(self.point()):
-            if p:
-                p.destroy()
-        for a in self.parents():
+        if self.point()[0]: self.point()[0].destroy()
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)